src/App.css

```diff
@@ -54,7 +54,7 @@ body {
     /* height: auto; */
 
 
-.NavBar {
+.nav-bar {
```

Comment:
Conventions are useful for (1) reducing confusion when reading and (2) reducing decision friction. If there's always a "right" way to write a css class, you just do that and move on.

The most common convention for css class names is to use kebab case, as I've changed it here.